lib/index.js

```diff
@@ -19,8 +18,4 @@ parser.registerSchemaParser([
   'application/vnd.oai.openapi+yaml;version=3.0.0',
 ], openapiSchemaParser);
 
-parser.registerSchemaParser([
-  'application/raml+yaml;version=1.0',
-], ramlDtParser);
-
```

Comment:
I can understand why we move the functionality for parsing RAML. However why do we remove the parser completely :thinking: ?

Reply:
@jonaslagoni this is optional parser, so doesn't have to be here, it actually makes a lot sense to have all the additional parsers used as plugins. Look on readme here how you would use it https://github.com/asyncapi/raml-dt-schema-parser/

This change makes the parser very nice as you can decide not to use our parser for RAML but write your own and plug it in. Plus, the bundle size is getting smaller as you have in it only the required minimum

@fmvilas it shoudl definitely happen with openapi parser too, please do not forget to create issue.

Reply:
Alright understandable, do we have/need an issue in the generator which includes this new parser 🤔?

Reply:
What new parser, @jonaslagoni?

Reply:
The optional RAML Schema parser that is being separated, I would imagine we had to include that [here ](https://github.com/asyncapi/generator/blob/master/lib/generator.js#L209) right?

Reply:
Oh, good point! You're right. Doing it now. Thanks, mate!

Reply:
Took the opportunity to extract the OpenAPI schema parser too: github.com/asyncapi/openapi-schema-parser.

Reply:
@jonaslagoni https://github.com/asyncapi/generator/pull/247 😄